CODING_STANDARDS.md

```diff
@@ -354,3 +396,13 @@ When it is not super obvious, include a brief description of what a class repres
 ```
 
 [ts-mixins]: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-2.html#support-for-mix-in-classes
+
+#### Prefer CSS classes to tag names for styling
+TODO(wagnermaciel): Ask @jelbourn why we prefer this.
```

Comment:
One of the explanations could be that targeting the tag name will cause a conflict with the MDC version of the component.